engine/settings/base.py

```diff
@@ -186,10 +219,10 @@ class DatabaseTypes:
             REDIS_URI,
         ],
         "OPTIONS": {
-            "DB": 1,
+            "DB": REDIS_DATABASE,
```

Comment:
Please take a close look at these lines of code: [REDIS_URI definition](https://github.com/grafana/oncall/blob/dev/engine/settings/base.py#L177-L179), [remote cache](https://github.com/grafana/oncall/blob/dev/engine/settings/base.py#L182-L200), [broker](https://github.com/grafana/oncall/blob/dev/engine/settings/base.py#L417-L422). In case of a Redis-based broker for celery, this project uses both `0` and `1` databases by default. There are two default databases, so I just chose one that is default for redis library.

But perhaps you are right, the project uses rabbitmq broker instead of redis by default, but uses redis cache with 1. Was there a reason to use a non-default value for library database number?

In addition, the default number `1` may confuse users because most part of projects use `0` when not set.

So what should we use as default then? :)

Reply:
I set the default REDIS_DATABASE to `1` for both broker and cache. But for helm-based installation, it is still `0` because the chart always defines this value.